src/snowflake/snowpark/modin/plugin/extensions/index.py

```diff
@@ -132,7 +133,7 @@ def __init__(
 
         Parameters
         ----------
-        data : array-like (1-dimensional), pandas.Index, modin.pandas.Series, optional
+        data : array-like (1-dimensional), pandas.Index, modin.pandas.Series, SnowflakeQueryCompiler, optional
```

Comment:
this is a change made last week, qc will be given throw argument `query_compiler`.
```suggestion
        data : array-like (1-dimensional), pandas.Index, modin.pandas.Series, optional
```